src/MSBuild/Resources/Strings.resx

```diff
@@ -1105,15 +1105,15 @@
       LOCALIZATION: The prefix "MSBUILD : error MSBxxxx:" should not be localized.
     </comment>
   </data>
-  <data name="MissingFeatureAvailabilityError" UESanitized="true" Visibility="Public">
```

Comment:
We probably shouldn't remove these metadata. Please edit the resx file manually to prevent unwanted changes.

Reply:
reverted